Jenkinsfile

```diff
@@ -4,12 +4,14 @@ import groovy.json.JsonOutput
 @Library('realm-ci') _
 repoName = 'realm-js' // This is a global variable
 
-def nodeVersions = ['8.15.0', '10.15.1']
-def electronVersions = ['2.0.18', '3.0.16', '3.1.8', '4.0.8', '4.1.4', '4.2.6']
+def nodeVersions = ['10.18.1']
```

Comment:
it's not needed. we should be testing for the lowest supported platform version only, since it's where we might have problems. new versions does not break the API only add to it.

Reply:
🤞